scripts/publish-desktop-release-assets.ps1

```
@@ -0,0 +1,122 @@
+
+$latestTag = ((& gh api "repos/$Repository/releases/latest" '--jq' '.tag_name') | Out-String).Trim()
+if ($LASTEXITCODE -ne 0 -or $latestTag -ne $Tag) {
+    throw "Tag $Tag must be the current GitHub stable release before registering its update metadata"
+}
+
+$ossRoot = $OssReleaseRoot.TrimEnd('/')
+$cdnRoot = $CdnBaseUrl.TrimEnd('/')
+foreach ($asset in $assets) {
+    Write-Host "Uploading staged asset $($asset.Name)"
+    $objectUrl = '{0}/{1}/{2}/{3}' -f $ossRoot, $Product, $version, $asset.Name
+    Invoke-Checked ossutil 'cp' $asset.FullName $objectUrl '-f' '--meta' 'Cache-Control:public, max-age=31536000, immutable'
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Invoke-Checked' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)